04-binary-data/lab-04/tests/index.test.js

```diff
@@ -1,39 +1,199 @@
 'use script';
 
-
 const readFile = require('../libs/read-file.js');
+const writeFile = require('../libs/write-file.js');
+const Bitmap = require('../libs/bmp-parse.js');
+const makeGreen = require('../libs/green.js');
+const makeNegative = require('../libs/negative.js')
+const makeVisor = require('../libs/makeVisor.js');
 
+const nameArg = '../assets/baldy.bmp';
+const path = `${__dirname}/${nameArg}`;
 
+describe('Test to verify file I/O for read and write ', () => {
 
-xdescribe('Test to verify that CLI arguments can be reveived', () => { });
-
-describe('Test to verify a file can be read ', () => {
+  it('should successfully load a file into the buffer', (done) => {
 
-  it('should successfully load a file into the buffer', () => {
+    readFile(`${__dirname}/../assets/baldy.bmp`, (err, actual) => {
+      if (err) throw console.error(err);
+      let expected = 0;
+      expect(actual.toString().length).not.toBe(expected);

```

Comment:
jest has a method called .toBeGreaterThan() that might be useful in a situation like this. for example: `expect(actual.toString().length).toBeGreaterThan(0)`;